### ISSUE TITLE
Windows image building

### ISSUE THREAD

**Issue by BryanQuigley** [canonical/packer-maas#3]
It would be nice to be able to build Windows MAAS images as well.

Feel free to put partial notes here if anyone is working on this.

**Comment by urskog84** [comment]
+1
Can any one make an example json-template?

Free evaluation ISO-file to win2019 can be found at

"iso_url": "http://download.microsoft.com/download/1/4/9/149D5452-9B29-4274-B6B3-5361DBDA30BC/14393.0.161119-1705.RS1_REFRESH_SERVER_EVAL_X64FRE_EN-US.ISO"

**Comment by nniehoff** [comment]
I have a working template that works with Windows 2019 deploying to VMs.  However, I don't have access to any reasonably decent equipment to test deploying bare-metal.  Let me clean some stuff up and generate a readme and I'll post what I have but it might be next week.

**Comment by peterwi** [comment]
@nniehoff 
I can test on bare metal if you could share the template

**Comment by nniehoff** [comment]
@peterwi 
Thanks for the offer, I had a switch die in my lab which has prevented me from accessing my test setup so this was put on hold.  I was able to get access to some bare metal systems and deploy my image however there appears to be an issue with UEFI boot at the moment.  This may potentially be related to a couple of issues with MAAS:

https://bugs.launchpad.net/maas/+bug/1865515
https://bugs.launchpad.net/maas/+bug/1879012

My replacement switch should arrive early next week and I'll be able to continue testing.  I'll also be able to share an example template at that point.

**Comment by peterwi** [comment]
Hi @nniehoff 
Please let me know if you can share the template.
Thanks

**Comment by DjairdaSilva** [comment]
Hello team, do we have any estimative to have the templates to Windows Server?

**Comment by tldale** [comment]
Anything on this? i'd love to try it!

**Comment by brunobernardi** [comment]
Hello all, we have any estimative to have the templates to Windows ? 
it would be very useful for all maas users

**Comment by DjairdaSilva** [comment]
Looks like the most challenging for creating a Windows template is the drivers pack. 
In my case, I have a lot of servers providers like, supermicro, HP and IBM. I'm not sure how we can create a Windows image that will be compatible with all network cards for all bare-metal providers. But I'll love it if the Maas team creates an example template for us and share it.

**Comment by tldale** [comment]
I'd love to just get a template so I can get it working and set an unattended.xml. For my use case all of the drivers are already present in the default windows at least the network card is and it can download the others with windows update.

**Comment by speedyrazor** [comment]
I am looking at the fork of this for windows templates. Could someone post an example fo how to get this running please?

**Comment by abduansari** [comment]
I am also looking for building Windows image for MAAS. If any one has build for windows please help with the template

**Comment by alanbach** [comment]
There is a new pull request that adds universal Windows support. I'd appreciate if anyone can take a look and test it.

**Comment by alexsander-souza** [comment]
the Windows template has been merged to `main`

**Comment by nics90** [comment]
@narindergupta , @alexsander-souza : Tried to built windows ISO from the templates pushed into the latest repo but didn't work:

make windows ISO=/root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso VERSION=2019 HEADLESS=true

rm -f -rf output-* windows.tar.gz http/Autounattend.xml \
drivers.iso OVMF_VARS.fd
cp -v /usr/share/OVMF/OVMF_VARS_4M.fd OVMF_VARS.fd
'/usr/share/OVMF/OVMF_VARS_4M.fd' -> 'OVMF_VARS.fd'
mkisofs -o drivers.iso -V 'Extra Drivers' -input-charset utf-8 drivers
Total translation table size: 0
Total rockridge attributes bytes: 0
Total directory bytes: 2198
Path table size(bytes): 22
Max brk space used 0
176 extents written (0 MB)
sed s#@VERSION@#"Windows Server 2019 SERVERSTANDARD"#g http/Autounattend.xml.uefi.template > http/Autounattend.xml
sed -i s#@PKEY@#valid-key#g http/Autounattend.xml
sed -i 's//<\/ProductKey>/' http/Autounattend.xml
packer init . && packer build -var iso_path=/root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso \
                -var headless=true \
                -var ovmf_suffix= \
                windows.pkr.hcl
2024/09/10 12:20:16 [INFO] Packer version: 1.11.2 [go1.21.12 linux amd64]
2024/09/10 12:20:16 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/10 12:20:16 [INFO] PACKER_CONFIG env var set; attempting to open config file: /root/.packerconfig
2024/09/10 12:20:16 [WARN] Config file doesn't exist: /root/.packerconfig
2024/09/10 12:20:16 [INFO] Setting cache directory: /root/.cache/packer
2024/09/10 12:20:16 [TRACE] init: plugingetter.ListInstallationsOptions{PluginDirectory:"/root/.config/packer/plugins", BinaryInstallationOptions:plugingetter.BinaryInstallationOptions{APIVersionMajor:"5", APIVersionMinor:"0", OS:"linux", ARCH:"amd64", Ext:"", Checksummers:[]plugingetter.Checksummer{plugingetter.Checksummer{Type:"sha256", Hash:(*sha256.digest)(0xc00078e000)}}, ReleasesOnly:true}}
2024/09/10 12:20:16 [TRACE] listing potential installations for "github.com/hashicorp/qemu" that match "~> 1.0". plugingetter.ListInstallationsOptions{PluginDirectory:"/root/.config/packer/plugins", BinaryInstallationOptions:plugingetter.BinaryInstallationOptions{APIVersionMajor:"5", APIVersionMinor:"0", OS:"linux", ARCH:"amd64", Ext:"", Checksummers:[]plugingetter.Checksummer{plugingetter.Checksummer{Type:"sha256", Hash:(*sha256.digest)(0xc00078e000)}}, ReleasesOnly:true}}
2024/09/10 12:20:16 [INFO] (telemetry) Finalizing.
2024/09/10 12:20:16 waiting for all plugin processes to complete...
2024/09/10 12:20:17 [INFO] Packer version: 1.11.2 [go1.21.12 linux amd64]
2024/09/10 12:20:17 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/10 12:20:17 [INFO] PACKER_CONFIG env var set; attempting to open config file: /root/.packerconfig
2024/09/10 12:20:17 [WARN] Config file doesn't exist: /root/.packerconfig
2024/09/10 12:20:17 [INFO] Setting cache directory: /root/.cache/packer
2024/09/10 12:20:17 [TRACE] listing potential installations for "github.com/hashicorp/qemu" that match "~> 1.0". plugingetter.ListInstallationsOptions{PluginDirectory:"/root/.config/packer/plugins", BinaryInstallationOptions:plugingetter.BinaryInstallationOptions{APIVersionMajor:"5", APIVersionMinor:"0", OS:"linux", ARCH:"amd64", Ext:"", Checksummers:[]plugingetter.Checksummer{plugingetter.Checksummer{Type:"sha256", Hash:(*sha256.digest)(0xc0007ae000)}}, ReleasesOnly:false}}
2024/09/10 12:20:17 [TRACE] Found the following "github.com/hashicorp/qemu" installations: [{/root/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 v1.1.0 x5.0}]
2024/09/10 12:20:17 [INFO] found external [-packer-default-plugin-name-] builders from qemu plugin
2024/09/10 12:20:17 [TRACE] listing potential installations for <nil> that match "". plugingetter.ListInstallationsOptions{PluginDirectory:"/root/.config/packer/plugins", BinaryInstallationOptions:plugingetter.BinaryInstallationOptions{APIVersionMajor:"5", APIVersionMinor:"0", OS:"linux", ARCH:"amd64", Ext:"", Checksummers:[]plugingetter.Checksummer{plugingetter.Checksummer{Type:"sha256", Hash:(*sha256.digest)(0xc0007ae080)}}, ReleasesOnly:false}}
2024/09/10 12:20:17 [INFO] found external [-packer-default-plugin-name-] builders from qemu plugin
2024/09/10 12:20:17 [TRACE] validateValue: not active for headless, so skipping
2024/09/10 12:20:17 [TRACE] validateValue: not active for iso_path, so skipping
2024/09/10 12:20:17 [TRACE] validateValue: not active for ovmf_suffix, so skipping
2024/09/10 12:20:17 [TRACE] validateValue: not active for filename, so skipping
2024/09/10 12:20:17 [INFO] Starting external plugin /root/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 start builder -packer-default-plugin-name-
2024/09/10 12:20:17 Starting plugin: /root/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 []string{"/root/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64", "start", "builder", "-packer-default-plugin-name-"}
2024/09/10 12:20:17 Waiting for RPC address for: /root/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Plugin address: unix /tmp/packer-plugin3026581078
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Waiting for connection...
2024/09/10 12:20:17 Received unix RPC address for /root/.config/packer/plugins/github.com/hashicorp/qemu/packer-plugin-qemu_v1.1.0_x5.0_linux_amd64: addr is /tmp/packer-plugin3026581078
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Serving a plugin connection...
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 [TRACE] starting builder -packer-default-plugin-name-
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 use specified accelerator: kvm
2024/09/10 12:20:17 [INFO] Starting internal plugin packer-post-processor-shell-local
2024/09/10 12:20:17 Starting plugin: /usr/bin/packer []string{"/usr/bin/packer", "execute", "packer-post-processor-shell-local"}
2024/09/10 12:20:17 Waiting for RPC address for: /usr/bin/packer
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: [INFO] Packer version: 1.11.2 [go1.21.12 linux amd64]
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: /root/.packerconfig
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: [WARN] Config file doesn't exist: /root/.packerconfig
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: [INFO] Setting cache directory: /root/.cache/packer
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: args: []string{"packer-post-processor-shell-local"}
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: Plugin address: unix /tmp/packer-plugin3351527693
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: Waiting for connection...
2024/09/10 12:20:17 Received unix RPC address for /usr/bin/packer: addr is /tmp/packer-plugin3351527693
2024/09/10 12:20:17 packer-post-processor-shell-local plugin: Serving a plugin connection...
2024/09/10 12:20:17 [INFO] Starting internal plugin packer-post-processor-compress
2024/09/10 12:20:17 Starting plugin: /usr/bin/packer []string{"/usr/bin/packer", "execute", "packer-post-processor-compress"}
2024/09/10 12:20:17 Waiting for RPC address for: /usr/bin/packer
2024/09/10 12:20:17 packer-post-processor-compress plugin: [INFO] Packer version: 1.11.2 [go1.21.12 linux amd64]
2024/09/10 12:20:17 packer-post-processor-compress plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2024/09/10 12:20:17 packer-post-processor-compress plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: /root/.packerconfig
2024/09/10 12:20:17 packer-post-processor-compress plugin: [WARN] Config file doesn't exist: /root/.packerconfig
2024/09/10 12:20:17 packer-post-processor-compress plugin: [INFO] Setting cache directory: /root/.cache/packer
2024/09/10 12:20:17 packer-post-processor-compress plugin: args: []string{"packer-post-processor-compress"}
2024/09/10 12:20:17 packer-post-processor-compress plugin: Plugin address: unix /tmp/packer-plugin2164800799
2024/09/10 12:20:17 packer-post-processor-compress plugin: Waiting for connection...
2024/09/10 12:20:17 Received unix RPC address for /usr/bin/packer: addr is /tmp/packer-plugin2164800799
2024/09/10 12:20:17 packer-post-processor-compress plugin: Serving a plugin connection...
a checksum is highly recommended.

  on windows.pkr.hcl line 33:
  (source code not available)

2024/09/10 12:20:17 Build debug mode: false
2024/09/10 12:20:17 Force build: false
2024/09/10 12:20:17 On error:
2024/09/10 12:20:17 Waiting on builds to complete...
2024/09/10 12:20:17 Starting build run: qemu.windows_builder
2024/09/10 12:20:17 Running builder: qemu
2024/09/10 12:20:17 [INFO] (telemetry) Starting builder qemu.windows_builder
Warning: A checksum of 'none' was specified. Since ISO files are so big,
a checksum is highly recommended.

  on windows.pkr.hcl line 33:
  (source code not available)


qemu.windows_builder: output will be in this color.

2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Qemu path: /usr/bin/qemu-system-x86_64, Qemu Image path: /usr/bin/qemu-img
==> qemu.windows_builder: Retrieving ISO
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Acquiring lock for: /root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso (/root/.cache/packer/d09b17e2871977097a6c2ad8a0fc25fd1fe0ef02.iso.lock)
==> qemu.windows_builder: Trying /root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso
==> qemu.windows_builder: Trying /root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso
==> qemu.windows_builder: /root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso => /root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Leaving retrieve loop for ISO
==> qemu.windows_builder: Creating floppy disk...
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Floppy label is set to flop
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Floppy path: /tmp/packer1475562672
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Initializing block device backed by temporary file
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Formatting the block device with a FAT filesystem...
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Initializing FAT filesystem on block device
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Reading the root directory from the filesystem
    qemu.windows_builder: Copying files flatly from floppy_files
    qemu.windows_builder: Copying file: ./http/Autounattend.xml
    qemu.windows_builder: Copying file: ./http/logon.ps1
    qemu.windows_builder: Copying file: ./http/rh.cer
    qemu.windows_builder: Done copying files from floppy_files
    qemu.windows_builder: Collecting paths from floppy_dirs
    qemu.windows_builder: Resulting paths from floppy_dirs : []
    qemu.windows_builder: Done copying paths from floppy_dirs
    qemu.windows_builder: Copying files from floppy_content
    qemu.windows_builder: Done copying files from floppy_content
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 No CD files specified. CD disk will not be made.
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 [INFO] Creating disk with Path: output-windows_builder/packer-windows_builder and Size: 20G
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Executing qemu-img: []string{"create", "-f", "raw", "output-windows_builder/packer-windows_builder", "20G"}
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 stdout: Formatting 'output-windows_builder/packer-windows_builder', fmt=raw size=21474836480
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 stderr:
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Found available port: 8958 on IP: 0.0.0.0
==> qemu.windows_builder: Starting HTTP server on port 8958
    qemu.windows_builder: No communicator is set; skipping port forwarding setup.
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Looking for available port between 5900 and 6000 on 0.0.0.0
==> qemu.windows_builder: Looking for available port between 5900 and 6000 on 0.0.0.0
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Found available port: 5978 on IP: 0.0.0.0
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Found available VNC port: 5978 on IP: 0.0.0.0
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Qemu --version output: QEMU emulator version 6.2.0 (Debian 1:6.2+dfsg-2ubuntu6.22)
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: Copyright (c) 2003-2021 Fabrice Bellard and the QEMU Project developers
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Qemu version: 6.2.0
==> qemu.windows_builder: Starting VM, booting from CD-ROM
    qemu.windows_builder: view the screen of the VM, connect via VNC without a password to
    qemu.windows_builder: vnc://0.0.0.0:5978
    qemu.windows_builder: The VM will be run headless, without a GUI. If you want to
    qemu.windows_builder: view the screen of the VM, connect via VNC without a password to
    qemu.windows_builder: vnc://0.0.0.0:5978
==> qemu.windows_builder: Overriding default Qemu arguments with qemuargs template option...
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Executing /usr/bin/qemu-system-x86_64: []string{"-boot", "d", "-smp", "2", "-fda", "/tmp/packer1475562672", "-vnc", "0.0.0.0:78", "-device", "e1000,netdev=user.0", "-cdrom", "/root/17763.3650.221105-1748.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso", "-name", "packer-windows_builder", "-drive", "if=pflash,format=raw,id=ovmf_code,readonly=on,file=/usr/share/OVMF/OVMF_CODE.fd", "-drive", "if=pflash,format=raw,id=ovmf_vars,file=OVMF_VARS.fd", "-drive", "file=output-windows_builder/packer-windows_builder,format=raw", "-drive", "file=drivers.iso,media=cdrom,index=3", "-netdev", "user,id=user.0", "-m", "4096M", "-machine", "type=q35,accel=kvm", "-serial", "stdio", "-cpu", "host"}
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Started Qemu. Pid: 976429
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Qemu stderr: WARNING: Image format was not specified for '/tmp/packer1475562672' and probing guessed raw.
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Qemu stderr:          Automatically detecting the format is dangerous for raw images, write operations on block 0 will be restricted.
2024/09/10 12:20:17 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:17 Qemu stderr:          Specify the 'raw' format explicitly to remove the restrictions.
==> qemu.windows_builder: Waiting 1s for boot...
==> qemu.windows_builder: Connecting to VM via VNC (0.0.0.0:5978)
2024/09/10 12:20:20 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:20 Connected to VNC desktop: QEMU (packer-windows_builder)
==> qemu.windows_builder: Typing the boot commands over VNC...
2024/09/10 12:20:20 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:20 Special code '<enter>' found, replacing with: 0xFF0D
2024/09/10 12:20:20 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:20 [INFO] communicator disabled, will not connect
2024/09/10 12:20:20 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:20 Unable to load communicator config from state to populate provisionHookData
2024/09/10 12:20:20 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:20 Running the provision hook
    qemu.windows_builder: No communicator is configured -- skipping StepWaitGuestAddress
==> qemu.windows_builder: Waiting for shutdown...
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: !!!! X64 Exception Type - 0D(#GP - General Protection)  CPU Apic ID - 00000000 !!!!
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: ExceptionData - 0000000000000000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: RIP  - 000000007FA6CBC6, CS  - 0000000000000038, RFLAGS - 0000000000010002
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: RAX  - 49C7C83113FA7698, RCX - 0000000000000000, RDX - 000000007F92E018
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: RBX  - 0000000000000020, RSP - 000000007FF00990, RBP - 00000000746E7684
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: RSI  - 00000000746E7668, RDI - 000000007F92E018
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: R8   - 8000000000000001, R9  - 000000007FF00AC0, R10 - 0000000000000000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: R11  - 0000000000001000, R12 - 00000000746E7665, R13 - 00000000FFE00000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: R14  - 0000000000000000, R15 - 000000007F436818
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: DS   - 0000000000000030, ES  - 0000000000000030, FS  - 0000000000000030
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: GS   - 0000000000000030, SS  - 0000000000000030
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: CR0  - 0000000080010033, CR2 - 0000000000000000, CR3 - 000000007FC01000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: CR4  - 0000000000000668, CR8 - 0000000000000000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: DR0  - 0000000000000000, DR1 - 0000000000000000, DR2 - 0000000000000000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: DR3  - 0000000000000000, DR6 - 00000000FFFF0FF0, DR7 - 0000000000000400
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: GDTR - 000000007F9DE000 0000000000000047, LDTR - 0000000000000000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: IDTR - 000000007F470018 0000000000000FFF,   TR - 0000000000000000
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: FXSAVE_STATE - 000000007FF005F0
2024/09/10 12:20:23 packer-plugin-qemu_v1.1.0_x5.0_linux_amd64 plugin: 2024/09/10 12:20:23 Qemu stdout: !!!! Find image based on IP(0x7FA6CBC6) (No PDB)  (ImageBase=000000007FA67000, EntryPoint=000000007FAA7503) !!!!
